### PR TITLE
Fix an issue to support unreliable in GenericTest

### PR DESCRIPTION
**Pull request by IcePigZDB**
A possible solution to #204 and #187. The idea is make `partitioner` function support both `unreliable` and`partitions`.

### CODE REVIEW

File: kv/test_raftstore/test_test.go [hunk 1]
```diff
@@ -236,10 +239,14 @@ func GenericTest(t *testing.T, part string, nclients int, unreliable bool, crash
 			}
 		})
 
-		if partitions {
+		// enable cases:
+		//               partitions    no partitions
+		//    unreliable       1                1
+		// no unreliable       1                1
```
Comment (Connor1996) [inline kv/test_raftstore/test_test.go]: what's the meaning of 1?

Comment (IcePigZDB) [reply kv/test_raftstore/test_test.go]: It is a support matrix, 1 stands for support. It is used to show supported cases.

Comment (Connor1996) [reply kv/test_raftstore/test_test.go]: don't think it provides any useful information, let's remove it.

File: kv/test_raftstore/test_test.go [hunk 2]
```diff
@@ -98,28 +98,31 @@ func checkConcurrentAppends(t *testing.T, v string, counts []int) {
 }
 
 // repartition the servers periodically
-func partitioner(t *testing.T, cluster *Cluster, ch chan bool, done *int32, unreliable bool, electionTimeout time.Duration) {
+func partitioner(t *testing.T, cluster *Cluster, ch chan bool, done *int32, unreliable bool, partitions bool, electionTimeout time.Duration) {
```
Comment (Connor1996) [inline kv/test_raftstore/test_test.go]: how about renaming the `partitioner` to `networkchaos` or something else

Comment (IcePigZDB) [reply kv/test_raftstore/test_test.go]: Sorry to resp so late, your advice souds great.

**Review by Connor1996 — APPROVED** [review]
LGTM

**Comment by Connor1996** [comment]
@IcePigZDB Please sign the CLA

**Comment by IcePigZDB** [comment]
> @IcePigZDB Please sign the CLA
CLA is ok, how can I re run the CLA check?

**Comment by IcePigZDB** [comment]
> [CLA assistant check](https://cla-assistant.io/tidb-incubator/tinykv?pullRequest=265)
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/tidb-incubator/tinykv?pullRequest=265) before we can accept your contribution.
> **1** out of **2** committers have signed the CLA.
> 
> ✅ IcePigZDB
> ❌ Connor1996
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tidb-incubator/tinykv?pullRequest=265) it.

@Connor1996  I find it, it need your CLA too.

**Pull request merged**